sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3780,6 +3779,11 @@ object GpuOverrides extends Logging {
             case _ => false
           })
 
+        def hasDateTimeType(dt: DataType): Boolean =
```

Comment:
We need to look at how to update the json scan too. We might also want to add a psNote on the jsonStructReadTypes so that Date and Timestamp have notes on them that they are not supported by default.